content/docs/user-guide/networking/multiplayer/autocomponents.md

```diff
@@ -107,3 +107,34 @@ set(FILES
     Source/AutoGen/MySimpleNetPlayerComponent.AutoComponent.xml    
 )    
 ```
+
+## Network inputs
+
+Multiplayer auto-components can have *network inputs*, which are used to send input data from the player to the authoritative server. Network inputs are like special RPCs that the Multiplayer Gem creates, processes, and records at each frame of the network tick. When the authority receives a network input from the player, it stamps the network input with a frame number. Noting the frame number allows the authority to rewind, meaning it returns all the rewindable network properties to that moment in history, before it processes the input. Rewind is crucial to keep the multiplayer simulation in sync.
+
+### Attributes
+
+| Property | Description |
+| --- | --- |
+| **Name** | The name of the input. This name is used when writing or reading the input in code. |
+| **Type** | The data type that the input is stored as. |
+| **Init** | The initial value of the data. |
+
+### Example
+
+```xml
+<NetworkInput Type="StickAxis" Name="ForwardAxis" Init="0.0f" />
+<NetworkInput Type="bool"      Name="Crouch"      Init="false" />
+<NetworkInput Type="uint8_t"   Name="ResetCount"  Init="0" />
+```
+
+For a complete example, refer to [`NetworkPlayerMovementComponent.AutoComponent.xml`](https://github.com/o3de/o3de-multiplayersample/blob/development/Gem/Code/Source/AutoGen/NetworkPlayerMovementComponent.AutoComponent.xml#L21-L28).
+
+### Using network inputs in game logic
+
+In C++ and scripting, an auto-component with a network input overrides the following controllers:
+
+- `CreateInput`: The multiplayer system automatically calls `CreateInput` for the autonomous player at every network tick.
+
+- `ProcessInput`: This function calls for both the autonomous player and the authority.
```

Comment:
Important to note that ProcessInput is when you act upon all the input. Up until now we've been recording key presses, but not doing anything with it that affects the world. So if 'W' was pressed, now is the time to move the player forward.

Reply:
Revised this with your explanation
```suggestion
- `ProcessInput`: Use this function to process all of the network inputs. Prior to calling this function, you should have only recorded device inputs by calling CreateInput, which doesn't result in any changes to the world yet. When you call ProcessInput, both the server and client-player will process the same network input at the same network tick. This function calls for both the autonomous player and the authority.
```